packages/atomic/playwright-utils/cache.ts

```diff
@@ -0,0 +1,40 @@
+import {APIResponse, Request, Route} from '@playwright/test';
+import {createHash} from 'crypto';
+import {existsSync, mkdirSync, readFileSync, writeFileSync} from 'fs';
+import {join} from 'path';
+
+export const cacheFilePath = './playwright/.cache/';
+
+export const hashRequest = (url: string, body: string | null) => {
+  const str = url + body;
+  return createHash('sha512').update(str).digest('hex');
+};
+
+export const cache = async (key: string, response: APIResponse) => {
+  mkdirSync(cacheFilePath, {recursive: true});
+  const body = await response.text();
+  // TODO: check if the file already exists and throw
+  writeFileSync(join(cacheFilePath, key), body);
+  return body;
+};
+
+export const cacheHandler = async (route: Route, request: Request) => {
+  console.log('intercepted ---', request.url());
+  const {clientId, ...rest} = request.postDataJSON();
```

Comment:
If we want to handle search API requests too, I think we could do:

```typescript
const {clientId, analytics, ...rest} = request.postDataJSON();
```

although this might be too broad...

another option would be to filter out non-static params case-by-case, based on the request.url